tests/sflow/test_sflow.py

```diff
@@ -114,7 +114,7 @@ def get_ifindex(duthost, port):
 
 def get_port_index(duthost, port):
     py_version = 'python' if '201911' in duthost.os_version else 'python3'
```

Comment:
>201911 [](http://example.com/codeflow?start=30&length=6)

If the code is testing an old image (such as 201911 branch image), will it break?